pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -172,6 +172,33 @@ Once complete, continue to [attach your cluster][attach-cluster] to Kommander.
 
 You should expect to see that cert-manager will fail to deploy due to your existing `cert-manager` installation. This is expected and can be ignored.
 
+### Kommander Cluster with custom SSL certificate
+
+When attaching a cluster, it is expected that the managed cluster was deploying apps as federated by the management cluster.
+If the management cluster was initialised using a custom SSL certificate, the managed cluster may fail cloning the managers’ service repository. You may need to check the status of the federated GIT repository resource to see the error;
+
+```
+kubectl get gitrepo -A --kubeconfig MANAGED-KUBECONFIG
+[..]
+unable to clone 'https://MANAGER_INGRESS_ADDRESS/dkp/kommander/git/kommander/kommander': Get "https://MANAGER_INGRESS_ADDRESS/dkp/kommander/git/kommander/kommander/info/refs?service=git-upload-pack": x509: certificate signed by unknown authority
+[..]
+```
+
+The deployment fails because the managed cluster uses the wrong CA certificate to verify access to the management clusters’ GIT repository. For working around this issue we need to patch the `gitserver-ca` secret within the `kommander-flux` namespace on the managed cluster with the CA certificate as stored in the `kommander-traefik-certificate` secret within the `kommander`namespace on the management cluster.
+
+```
+kubectl --kubeconfig=MANAGED_KUBECONFIG patch secret -n kommander-flux gitserver-ca -p '{"data":{"caFile":"'$(kubectl --kubeconfig=MANAGER_KUBECONFIG get secret -n kommander kommander-traefik-certificate -o go-template='{{index .data "ca.crt"}}')'"}}'
+```
+
+You may need to trigger a reconciliation of the flux controller on the managed cluster if you do not want to wait for its regular interval to occur. Use the `flux` CLI utility -  https://fluxcd.io/docs/installation/ ;
```

Comment:
```suggestion
You may need to trigger a reconciliation of the flux controller on the managed cluster if you do not want to wait for its regular interval to occur. Use the [`flux` CLI utility][flux-cli]:
```
(need to add the `[flux-cli]` link at the bottom of this page.)

Reply:
Deliberately did this - and it was wrong :D 

So I was somehow assuming that we would try to make external links show as such. Turns out that is not our style at all. Check.

@makkes one more question on this:
Is it that you would prefer a link to the general project homepage instead of a deep-link into the installation section? 

Then on the linking in general, you are suggesting to use reference style links - without your example, I would have gone for "normal" links and again I would be wrong, not our style it seems - briefly checked our docs on that as well.

Reply:
There is one more thing I wonder about in general: is it a good idea to mix styles?

That is; having a \`code-block\` be embedded in a [link]. My first impulse here is: DONT. @gvasquezvargas is there something you would have as a guideline?

Reply:
Hi @tillt! I have not had the chance to learn about how we use links yet, let me ask and I will get back to you ;)